src/calculateRentalCost.js

```diff
@@ -1,10 +1,28 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_FOR_DAY = 40;
+  const LITTLE_DAYS = 3;
+  const LITTLE_DISCOUNT = 20;
+  const BIG_DAYS = 7;
+  const BIG_DISCOUNT = 50;
+
+  const TOTAL_COST = days * COST_FOR_DAY;
```

Comment:
```suggestion
  const totalCost = days * COST_FOR_DAY;
```
Variables should be in camelCase, only constants that are not dynamic values in UPPER_CASE